MusicStore.Web/SeedData.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Microsoft.AspNet.Identity;
+using Microsoft.AspNet.Identity.EntityFramework;
+using MusicStore.Web.Models;
+
+namespace MusicStore.Web
+{
+    public class SeedData
+    {
+        public static void Initialize(ApplicationDbContext modelContext)
+        {
+            if (!modelContext.Database.Exists())
+            {
+                throw new ArgumentException("Can not find dataBase, connection is wrong");
+            }
+
+            var userManager = new ApplicationUserManager(new UserStore<ApplicationUser>(modelContext));
+
+            var roleManager = new RoleManager<IdentityRole>(new RoleStore<IdentityRole>(modelContext));
+
+            // создаем пользователей
```

Comment:
Unnecessary comment
I can identify what you are doing just looking at code, so it is not necessary to do this here 
Please, apply this to all file